### PR TITLE
KOGITO-7053: SWF Viewer - Community Documentation

### PR DESCRIPTION
**Pull request by danielzhe**
**JIRA:** https://issues.redhat.com/browse/KOGITO-7053

**Description:** SWF Viewer - Community Documentation

<details>
<summary>
How to setup automatic cherry-pick to another branch ?
</summary>

The cherry-pick action allows to setup automatic cherry-pick from `main` to a specific branch.

To allow it, you will need to add the corresponding label with pattern: `backport-{RELEASE_BRANCH}`.  
For example, if a backport to branch `1.26.x` is needed, then the label should be `backport-1.26.x`.

Once the PR is merged, the action will retrieve the commit and cherry-pick it to the desired branch.

*NOTE: You can still add label after the merge and it should still be cherry-picked.*
</details>

### CODE REVIEW

File: serverlessworkflow/modules/ROOT/pages/release_notes.adoc [hunk 1]
```diff
@@ -1,21 +1,14 @@
 = New features on {page-component-display-version}
 :compat-mode!:
 
-== Notable changes
+* New default diagram canvas (currently only available in VSCode extension)
```
Comment (romartin) [inline serverlessworkflow/modules/ROOT/pages/release_notes.adoc:4]: @danielzhe I think this is not true. The new stunner editor is presented in all channels (VSCode, sandbox, online, chome, etc). The only thing is is only VSCode specific is the setting to swith back to Mermaid editor.
Can you please @ederign @paulovmr confirm this?
Thx in advance!

**Review by romartin — COMMENTED** [review]
Hey  @danielzhe ,

Thanks for this work, looks good!

Just a couple of things:
- I added a review comment for the first change you mentioned about only available in vscode, I think it's not true, but let's wait for @ederign @paulovmr to confirm
- About the other changes mentioned (like error handling, state selection, etc), I think we can expand a bit more the text, maybe it is not so clear for first time users. As I am also writting a blog post [1], I suggest you to look at my list of features and please, let me know your feebback in the doc too, and then we can just paste both texts in my doc also here, for documentation.

[1] The doc for the blog post is here, please feel free to comment 
https://docs.google.com/document/d/1gcDux4LgftIj81zFPY01nW4C7fOBB0XU0OuSilmTROs/edit?usp=sharing

Thanks!

File: serverlessworkflow/modules/ROOT/pages/release_notes.adoc (hunk 1 above)
Comment (ricardozanini) [reply serverlessworkflow/modules/ROOT/pages/release_notes.adoc:4]: Please, keep the release notes unchanged here. I'll open a new PR to update the release notes targeting 1.28 of Kogito, and we can also add a new sub-title for the editors. In that case, please keep the format the same as we have here with a linked KOGITO Jira targeting the Editor's version compatible with Kogito 1.28.

**Review by ricardozanini — CHANGES_REQUESTED** [review]

File: serverlessworkflow/modules/ROOT/pages/release_notes.adoc (hunk 1 above)
Comment (ederign) [reply serverlessworkflow/modules/ROOT/pages/release_notes.adoc:4]: @romartin correct. Except for Chrome Extension (@paulovmr will send a PR for it), all channels are default. Both on VS Code and Web Tools (kubesmarts) there is a way to switch back to mermaid.

Comment (ederign) [reply serverlessworkflow/modules/ROOT/pages/release_notes.adoc:4]: @danielzhe could you please fix this ?

**Review by ederign — CHANGES_REQUESTED** [review]
Please update the available channels. (I commented on the other thread)

File: serverlessworkflow/modules/ROOT/pages/release_notes.adoc (hunk 1 above)
Comment (danielzhe) [reply serverlessworkflow/modules/ROOT/pages/release_notes.adoc:4]: I reverted this change as requested by @ricardozanini in the comment bellow.

**Comment by ederign** [comment]
@danielzhe please let me know when you adjust based on Zanini's review and I'll approve the PR.

I'll update the screenshot of the home screen of Web Tools as soon as I fix them!

**Comment by danielzhe** [comment]
@ederign Reverted!

**Review by ederign — APPROVED** [review]

**Review by spolti — APPROVED** [review]

**Review by ricardozanini — APPROVED** [review]

**Comment by ricardozanini** [comment]
@danielzhe @ederign the release notes for Kogito 1.28 (DP2) is here: #208 

You can add the relevant JIRAs in there. Please just keep the same format. We don't need to distinguish between editors and runtimes. Everything is Kogito Serverless Workflow :)

**Comment by danielzhe** [comment]
@kiegroup/gatekeepers Merge, please!

**Comment by ederign** [comment]
Hi @hmanwani-rh ! Just a friendly ping. Do you have bandwidth to review this

**Review by romartin — APPROVED** [review]

**Review by radtriste — APPROVED** [review]

**Review by domhanak — APPROVED** [review]

**Pull request merged**